jadbio/client.py

```diff
@@ -244,7 +244,7 @@ def upload_file(self, file_id: int, pth_to_file: str):
         >>> client.upload_file(1234, "pth/to/file.csv")
         1234
         """
-        with open(pth_to_file, 'r') as f:
+        with open(pth_to_file, 'r', encoding='utf-8-sig') as f:
```

Comment:
See https://stackoverflow.com/questions/22459020/python-decode-utf-16-file-with-bom for more